filters/include/pcl/filters/impl/radius_outlier_removal.hpp

```diff
@@ -80,10 +80,10 @@ pcl::RadiusOutlierRemoval<PointT>::applyFilterIndices (std::vector<int> &indices
     int mean_k = min_pts_radius_ + 1;
     double nn_dists_max = search_radius_ * search_radius_;
 
-    for (std::vector<int>::const_iterator it = indices_->begin (); it != indices_->end (); ++it)
+    for (const auto& index : (*indices_))
```

Comment:
(in several places) Needless brackets around *indices_

Reply:
Yes, they are not necessary, but I added them for clarity and safety.